test/e2e/specs/site-editor/multi-entity-saving.spec.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site Editor - Multi-entity save flow', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.visitSiteEditor( {
+			postId: 'emptytheme//index',
+			postType: 'wp_template',
+		} );
+		await editor.canvas.click( 'body' );
+	} );
+
+	test( 'save flow should work as expected', async ( { editor, page } ) => {
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+			attributes: {
+				content: 'Testing',
+			},
+		} );
+
+		await expect(
+			page
+				.getByRole( 'region', { name: 'Editor top bar' } )
+				.getByRole( 'button', { name: 'Save' } )
+		).toBeEnabled();
+		await expect(
+			page
+				.getByRole( 'region', { name: 'Save panel' } )
+				.getByRole( 'button', { name: 'Open save panel' } )
+		).toBeVisible();
+
+		await editor.saveSiteEditorEntities();
```

Comment:
Should we delete template parts before/after the tests to avoid the changes to persist between tests?

Reply:
A good idea. Added the template cleanup to the tests since we only change the templates now.